packages/atomic/src/components/search-box-suggestions/suggestions-common.ts

```diff
@@ -0,0 +1,41 @@
+import {SearchBox} from '@coveo/headless';
+import {VNode} from '@stencil/core';
+import {buildCustomEvent} from '../../utils/event-utils';
+import {Bindings} from '../../utils/initialization-utils';
+
+export interface SearchBoxSuggestionElement {
+  value: string;
+  onClick(): void;
+  content: VNode;
```

Comment:
The type VNode works great in Stencil and maybe in other JSX frameworks, but I don't know if it's possible without those frameworks to provide a VNode.